### PR TITLE
Host docs

### LINKED ISSUE #59: Host the most recent version of the FakeItEasy docs on the FakeItEasy website

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #59

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,7 +97,7 @@ <h3>Additional Features</h3>
 
       <div class="navigation">
         <a id="button-quickstart" class="pure-button" href="http://fakeiteasy.readthedocs.io/en/stable/quickstart/">Quickstart</a>
-        <a id="button-docs" class="pure-button" href="http://fakeiteasy.readthedocs.io/en/stable/">Documentation</a>
+        <a id="button-docs" class="pure-button" href="docs/">Documentation</a>
```
[the remark this answers is not in the document]
Comment (thomaslevesque) [reply index.html:100]: Yeah, let's wait until we have at least some of the older docs before we change this link.

**Review by thomaslevesque — APPROVED** [review]
Nice! I approved so that I don't have to come back later, but as discussed, we should drop the commit that changes the link from the home page.

**Comment by blairconrad** [comment]
>  I approved so that I don't have to come back later

Who do you think I'm going to force to hit the "Merge button"?

**Comment by thomaslevesque** [comment]
Thanks, @blairconrad!

> Who do you think I'm going to force to hit the "Merge button"?

Yeah, I thought of this after I sent my comment...

**Comment by blairconrad** [comment]
Thank you!

**Pull request merged**